cvs diff: Diffing solenv
cvs diff: Diffing solenv/bin
cvs diff: Diffing solenv/bin/modules
cvs diff: Diffing solenv/bin/modules/PCVSLib
cvs diff: Diffing solenv/bin/modules/PCVSLib/lib
cvs diff: Diffing solenv/bin/modules/PCVSLib/lib/PCVSLib
cvs diff: Diffing solenv/bin/modules/PCVSLib/samples
cvs diff: Diffing solenv/bin/modules/PCVSLib/t
cvs diff: Diffing solenv/bin/modules/installer
cvs diff: Diffing solenv/bin/modules/installer/windows
cvs diff: Diffing solenv/bin/modules/packager
cvs diff: Diffing solenv/bin/modules/par2script
cvs diff: Diffing solenv/bin/modules/pre2par
cvs diff: Diffing solenv/config
cvs diff: Diffing solenv/inc
Index: solenv/inc/libs.mk
===================================================================
RCS file: /cvs/tools/solenv/inc/libs.mk,v
retrieving revision 1.124
retrieving revision 1.124.34.1
diff -p -u -u -p -b -w -B -r1.124 -r1.124.34.1
--- solenv/inc/libs.mk	10 Jan 2008 13:02:36 -0000	1.124
+++ solenv/inc/libs.mk	13 Feb 2008 12:20:37 -0000	1.124.34.1
@@ -155,6 +155,7 @@ FREETYPELIBST=$(STATIC) -lfreetype $(DYN
 .ENDIF
 PSPLIB=-lpsp$(OFFICEUPD)$(DLLPOSTFIX)
 TKLIB=-ltk$(OFFICEUPD)$(DLLPOSTFIX)
+LAYOUTLIB=-ltklayout$(OFFICEUPD)$(DLLPOSTFIX)
 SVTOOLLIB=-lsvt$(OFFICEUPD)$(DLLPOSTFIX)
 .IF "$(GUI)$(COM)"=="WNTGCC"
 XMLSECLIB=-lxmlsec1-1
@@ -450,6 +451,7 @@ CPPCANVASLIB=icppcanvas.lib
 AGGLIB=iagg.lib
 PSPLIB=apsp.lib
 TKLIB=itk.lib
+LAYOUTLIB=itklayout.lib
 SVXLLIB=svxl.lib
 FREETYPELIB=freetype.lib
 FREETYPELIBST=freetype.lib
cvs diff: Diffing solenv/inc/startup
cvs diff: Diffing solenv/inc/startup/AIX
cvs diff: Diffing solenv/inc/startup/FREEBSD
cvs diff: Diffing solenv/inc/startup/HPUX
cvs diff: Diffing solenv/inc/startup/IRIX
cvs diff: Diffing solenv/inc/startup/LINUX
cvs diff: Diffing solenv/inc/startup/MACOSX
cvs diff: Diffing solenv/inc/startup/NETBSD
cvs diff: Diffing solenv/inc/startup/SOLARIS
cvs diff: Diffing solenv/inc/startup/UNIX
cvs diff: Diffing solenv/inc/startup/wnt
cvs diff: Diffing solenv/prj
cvs diff: Diffing solenv/src
cvs diff: Diffing solenv/unxmacxp
cvs diff: Diffing solenv/unxmacxp/inc
